github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
You are correct that native format is good but in this case i dont want hour:mins:secs so i do not want to keep it as 'time' in the struct. I will however called the json description as hour. This will make the code readable and satisfy both our needs.

```
type GitIssues struct {
	Number    int    `json:"number"`
	Title     string `json:"title"`
	Link      string `json:"html_url"`
	Labels    []Label
	Assignees string `json:"login"`
	Milestone string `json:"milestone"`
	State     string `json:"state"`
	CreatedAt string `json:"created_at"`
	UpdatedAt string `json:"updated_at"`
	Repo      string `json:"repository_url"`
	Age       int    `json:"hour"` <---------------
}
```